src/macaron/policy_engine/prelude/helper_rules.dl

```diff
@@ -9,6 +9,9 @@
 .decl check_passed(component_id: number, check_name: symbol)
 check_passed(component_id, check_name) :- check_result(_, check_name, 1, component_id).
 
+.decl check_failed(component_id: number, check_name: symbol)
+check_failed(component_id, check_name) :- check_result(_, check_name, 0, component_id).
```

Comment:
Have you checked what the `check_result` fact would be, or if it is even produced in the first place, in case a check is skipped (e.g. due to the include/exclude check configuration)?

Reply:
The policy verification will fail if any `check_passed` or `check_failed` checks are excluded. Although I had not checked until you mentioned it here.

Reply:
I see. So, excluded checks will not produce any facts. Thanks.